specification/electronic-prescription-service-api.yaml

```diff
@@ -54,7 +55,9 @@ info:
     - [Personal Demographics Service - FHIR API](https://digital.nhs.uk/developer/api-catalogue/personal-demographics-service-fhir) - use this to obtain patient data
 
     ## API status and roadmap
-    To see our roadmap, or to suggest, comment or vote on features for this API, see our [interactive product backlog](https://nhs-digital-api-management.featureupvote.com/?tag=eps-fhir-api).
+    The immediate EPS development priority is to support the rollout of the next-gen FHIR APIs for primary care and secondary care prescribing and dispensing. Alongside this we will be working on integrating a new FHIR-compliant prescription tracker into the next gen APIs.
+    
+    To see suggest, comment or vote on features for this API, see our [interactive product backlog](https://nhs-digital-api-management.featureupvote.com/?tag=eps-fhir-api).
```

Comment:
"To see suggest, comment or vote on features for this API" should this be "To see our roadmap,..." as in the previous version ?

Reply:
Ive just removed 'see' from this as the interactive product backlog doesn't really have much to do with out roadmap at the moment tbh